### PR TITLE
Bump to 4.2.6

### PR DESCRIPTION
**Pull request by alberpilot**
4.2.5 bump

### CODE REVIEW

File: rpms/SPECS/4.2.6/wazuh-manager-4.2.6.spec [hunk 1]
```diff
@@ -830,6 +830,8 @@ rm -fr %{buildroot}
 
 
 %changelog
+* Mon Mar 25 2022 support <info@wazuh.com> - 4.2.6
```
Comment (DFolchA) [inline rpms/SPECS/4.2.6/wazuh-manager-4.2.6.spec:833]: Correct the date.

File: rpms/SPECS/4.2.6/wazuh-agent-4.2.6.spec [hunk 2]
```diff
@@ -576,6 +576,8 @@ rm -fr %{buildroot}
 
 
 %changelog
+* Mon Mar 25 2022 support <info@wazuh.com> - 4.2.6
```
Comment (DFolchA) [inline rpms/SPECS/4.2.6/wazuh-agent-4.2.6.spec:579]: Correct the date.

File: debs/SPECS/4.2.6/wazuh-manager/debian/changelog [hunk 3]
```diff
@@ -1,3 +1,9 @@
+wazuh-manager (4.2.6-RELEASE) stable; urgency=low
+
+  * More info: https://documentation.wazuh.com/current/release-notes/
+
+ -- Wazuh, Inc <info@wazuh.com>  Mon, 25 Mar 2022 16:47:07 +0000
```
Comment (DFolchA) [inline debs/SPECS/4.2.6/wazuh-manager/debian/changelog:5]: Correct the date.

File: debs/SPECS/4.2.6/wazuh-agent/debian/changelog [hunk 4]
```diff
@@ -1,3 +1,9 @@
+wazuh-agent (4.2.6-RELEASE) stable; urgency=low
+
+  * More info: https://documentation.wazuh.com/current/release-notes/
+
+ -- Wazuh, Inc <info@wazuh.com>  Mon, 25 Mar 2022 16:47:07 +0000
```
Comment (DFolchA) [inline debs/SPECS/4.2.6/wazuh-agent/debian/changelog:5]: Correct the date.

**Review by DFolchA — CHANGES_REQUESTED** [review]

File: debs/SPECS/4.2.6/wazuh-agent/debian/changelog (hunk 4 above)
Comment (alberpilot) [reply debs/SPECS/4.2.6/wazuh-agent/debian/changelog]: Done 6318a8a96caf59fd477451b2802b4f8c620b3eb6

File: debs/SPECS/4.2.6/wazuh-manager/debian/changelog (hunk 3 above)
Comment (alberpilot) [reply debs/SPECS/4.2.6/wazuh-manager/debian/changelog]: Done 6318a8a96caf59fd477451b2802b4f8c620b3eb6

File: rpms/SPECS/4.2.6/wazuh-agent-4.2.6.spec (hunk 2 above)
Comment (alberpilot) [reply rpms/SPECS/4.2.6/wazuh-agent-4.2.6.spec]: Done 6318a8a96caf59fd477451b2802b4f8c620b3eb6

File: rpms/SPECS/4.2.6/wazuh-manager-4.2.6.spec (hunk 1 above)
Comment (alberpilot) [reply rpms/SPECS/4.2.6/wazuh-manager-4.2.6.spec]: Done 6318a8a96caf59fd477451b2802b4f8c620b3eb6

**Review by DFolchA — APPROVED** [review]
LGTM

**Pull request merged**